components/clp-package-utils/clp_package_utils/scripts/del_archives.py

```diff
@@ -0,0 +1,110 @@
+import argparse
+import logging
+import pathlib
+import subprocess
+import sys
+
+from clp_package_utils.general import (
+    CLP_DEFAULT_CONFIG_FILE_RELATIVE_PATH,
+    dump_container_config,
+    generate_container_config,
+    generate_container_name,
+    generate_container_start_cmd,
+    get_clp_home,
+    JobType,
+    load_config_file,
+    validate_and_load_db_credentials_file,
+)
+
+# Setup logging
+# Create logger
+logger = logging.getLogger(__file__)
+logger.setLevel(logging.INFO)
+# Setup console logging
+logging_console_handler = logging.StreamHandler()
+logging_formatter = logging.Formatter("%(asctime)s [%(levelname)s] [%(name)s] %(message)s")
+logging_console_handler.setFormatter(logging_formatter)
+logger.addHandler(logging_console_handler)
+
+
+def main(argv):
+    clp_home = get_clp_home()
+    default_config_file_path = clp_home / CLP_DEFAULT_CONFIG_FILE_RELATIVE_PATH
+
+    args_parser = argparse.ArgumentParser(description="Prune the out-dated archives.")
+    args_parser.add_argument(
+        "--config",
+        "-c",
+        default=str(default_config_file_path),
+        help="CLP package configuration file.",
+    )
+    args_parser.add_argument(
+        "--begin-ts",
+        type=int,
+        default=0,
+        help="Time range filter lower-bound (inclusive) as milliseconds" " from the UNIX epoch.",
+    )
+    args_parser.add_argument(
+        "--end-ts",
+        type=int,
+        required=True,
+        help="Time range filter upper-bound (inclusive) as milliseconds" " from the UNIX epoch.",
+    )
+    parsed_args = args_parser.parse_args(argv[1:])
+
+    # Validate and load config file
+    try:
+        config_file_path = pathlib.Path(parsed_args.config)
+        clp_config = load_config_file(config_file_path, default_config_file_path, clp_home)
+        clp_config.validate_logs_dir()
+
+        # Validate and load necessary credentials
+        validate_and_load_db_credentials_file(clp_config, clp_home, False)
+    except:
+        logger.exception("Failed to load config.")
+        return -1
+
+    # Validate the input timestamp
+    begin_ts = parsed_args.begin_ts
+    end_ts = parsed_args.end_ts
+    if begin_ts > end_ts:
+        logger.error("begin-ts must be less than or equal to end-ts")
+        return -1
+    if end_ts < 0 or begin_ts < 0:
+        logger.error("begin_ts and end_ts must be positive.")
+        return -1
+
+    container_name = generate_container_name(JobType.DEL_ARCHIVE)
```

Comment:
I don't think we need to add a new job type for the sake of a container name, right?

Reply:
Currently generate_container_name only takes in a JobType enum as argument.

Do you have any suggestions

Reply:
Can we make `generate_container_name` take a string?